schema_editor/.jshintrc

```diff
@@ -3,7 +3,7 @@
   "browser": true,
   "esnext": true,
   "bitwise": true,
-  "camelcase": true,
+  "camelcase": false,
```

Comment:
This shouldn't be disabled app-wide. Where there are unavoidable non-camelcase variables you can surround the code in: `/* jshint camelcase: false */` ... `/* jshint camelcase: true */`